dist-git-8.0.1-to-8.1.1.patch

```diff
@@ -0,0 +1,51 @@
+From f0c9550eaf836121e3734be817f92e558fd56541 Mon Sep 17 00:00:00 2001
+From: Skyler Clark <skylerjaneclark@gmail.com>
+Date: Fri, 7 Jul 2023 09:22:35 -0400
+Subject: [PATCH] [release-4.13]Prepare for 8.1.1 release
+
+  This commit was generated by hack/pre-release.sh
+
+Signed-off-by: Skyler Clark <skylerjaneclark@gmail.com>
+---
+ Dockerfile       | 2 +-
+ render_templates | 4 ++--
+ 2 files changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/Dockerfile b/Dockerfile
+index c5b1bfe..39abdc0 100644
+--- a/Dockerfile
++++ b/Dockerfile
+@@ -24,7 +24,7 @@ LABEL com.redhat.openshift.versions="=v4.13"
+ LABEL com.redhat.delivery.backport=false
+ 
+ # This label gets replaced by render_templates, so it is safer to keep it on its own
+-LABEL version="v8.0.0"
++LABEL version="v8.1.1"
+ 
+ # This label maps to the brew build target
+ LABEL com.redhat.component="windows-machine-config-operator-bundle-container"
+diff --git a/render_templates b/render_templates
+index 971e48a..b386a01 100755
+--- a/render_templates
++++ b/render_templates
+@@ -45,7 +45,7 @@ update_csv() {
+     sed -i "s|REPLACE_IMAGE|${wmco_image_sha}|g" ${csv}
+     # Following command adds the replaces field in the CSV using the previous released version of the operator.
+     # TODO: Uncomment below for next minor release
+-    sed -i '/version: 8.0.1/i \  replaces: '${wmco}'.v'${previous_version}'' ${csv}
++    sed -i '/version: 8.1.1/i \  replaces: '${wmco}'.v'${previous_version}'' ${csv}
+     sed -i "s|REPLACE_DATE|`date "+%Y-%m-%dT%H:%M:%SZ"`|g" ${csv}
+     sed -i "/command:/d" ${csv}
+     awk '{print} sub(/- args:/,"- command:")' ${csv} > tmp && mv tmp ${csv}
+@@ -85,7 +85,7 @@ wmco_image_sha=`echo ${build_info} | jq -e .extra.image.index.pull[0]`
+ wmco="windows-machine-config-operator"
+ csv="manifests/${wmco}.clusterserviceversion.yaml"
+ # TODO: Uncomment below for next minor release
+-previous_version="8.0.0"
++previous_version="8.0.1"
```

Comment:
And previous version is now `8.0.1` but you started by replacing `v8.0.0`. This does not seem correct.

Reply:
so on the 4.13 branch I ran this from, the [WMCO version is 8.0.1](https://github.com/openshift/windows-machine-config-operator/blob/d0be5e812c3e3914d3cf3c464623e688673565fb/Makefile#L6), and this was a y-stream update, so the correct version would be 8.1.1 

I think The 8.0.0 you see at the top was there because of the comment that said it gets replaced by render-templates, but it never got replaced

The matching for the render-templates is done intentionally to match on any version in the right spot, since I've seen that these files can have incorrect versions in them that only get changed when doing the post-release work.

Reply:
>so on the 4.13 branch I ran this from, the [WMCO version is 8.0.1](https://github.com/openshift/windows-machine-config-operator/blob/d0be5e812c3e3914d3cf3c464623e688673565fb/Makefile#L6), and this was a y-stream update, so the correct version would be 8.1.1

That is incorrect. A Y stream update for `8.0.1` will be `8.1.0`.

>I think The 8.0.0 you see at the top was there because of the comment that said it gets replaced by render-templates, but it never got replaced

This is worrisome. Aren't we releasing 8.0.1? Why is it still at 8.0.0?

>I've seen that these files can have incorrect versions in them that only get changed when doing the post-release work.

Haven't we done the post release work for 8.0.0 which would have bumped it to 8.0.1?